web/src/components/storage/AddExistingDeviceMenu.tsx

```diff
@@ -90,7 +93,16 @@ export default function AddExistingDeviceMenu() {
               description={<MenuDeviceDescription device={device} />}
               onClick={() => modelHook.addDrive(device.name)}
             >
-              {deviceLabel(device)}
+              <Split hasGutter>
```

Comment:
I'd try removing this Split and placing the deviceLabel inside the Flex unless the spacing using there is too small

Reply:
I wanted some extra space between the first part and the list of systems.

Reply:
Ok